cmd/juju/commands/repl.go

```diff
@@ -52,6 +53,18 @@ Type "help" to see a list of available commands.
 Type "q" or ^D or ^C to quit.
```

Comment:
Update here?
`"q", "quite", "exit" ...`

Reply:
As discussed live, better to advertise fewer, simpler options. But we do also react as expected if they do type "exit" etc